src/modules/common/components/TokenField/TokenField.js

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import TokenAmount from '@token/fungible/components/tokenAmount';
+import MenuSelect, { MenuItem } from 'src/modules/wallet/components/MenuSelect';
+import { useTokensBalance } from '@token/fungible/hooks/queries';
+import TokenStyle from './TokenField.css';
+
+// eslint-disable-next-line import/prefer-default-export
+export const TokenField = ({ styles, onChange, value }) => {
```

Comment:
@soroushm this was from this pr https://github.com/LiskHQ/lisk-desktop/pull/4486#issue-1381974064 I would rename it on my own PR since I have made little modifications to it here including adding unit test